test/services/VatDetailsServiceSpec.scala

```diff
@@ -41,13 +41,13 @@ class VatDetailsServiceSpec extends ControllerBaseSpec {
       "#001"
     )
 
-    val payment: Payment = Payment(
+    val payments: Payments = Payments(Seq(Payment(
```

Comment:
Can more payment models be added to this sequence. With only one payment model being returned it doesn't truly test the ordering when it is being used in tests below

Reply:
done